src/components/Tab/TabItem/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+export interface TabItemProps {
+  name: string;
+  active: boolean;
+  icon?: React.ComponentType<any>;
+  onlyIcon?: boolean;
+  onClick: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+const TabItem: React.FC<TabItemProps> = ({
+  name,
+  active,
+  icon: Icon,
+  onlyIcon = false,
```

Comment:
Maybe we can use the `name` and `icon` props to know if we render one or the other, or both

The order is Icon and name, right?

We can do it
```
{icon ?? <Icon... />
{name ?? <Name ... />
```

This way we simplify the code

I suggest renaming Name to Title